### PR TITLE
Define goldstar-specific style recommendations

### PR DESCRIPTION
**Pull request by tlehman**
During our lunch-and-learn, we debated the merits of various equivalent ways of writing ruby code. The conversation is not over, it will continue in this pull request.

### CODE REVIEW

**Comment by jaredhoyt** [comment]
Will we do a pull request for each lunch and learn or will this live until we've made it all the way through the guide?

**Comment by tlehman** [comment]
I think until we make it through the guide, we can merge in our changes, and then open up new PRs if we miss something.

**Comment by therubymug** [comment]
What @tlehman said.

File: README.md [hunk 1]
```diff
@@ -363,24 +363,6 @@ Translations of the guide are available in the following languages:
...
-  # good (and a bit more width efficient)
-  kind =
-    case year
-    when 1850..1889 then 'Blues'
-    when 1890..1909 then 'Ragtime'
-    when 1910..1929 then 'New Orleans Jazz'
-    when 1930..1939 then 'Swing'
-    when 1940..1950 then 'Bebop'
-    else 'Jazz'
-    end
```
Comment (jescholl) [inline README.md:360]: This style of spacing is used later on in a section we already went over on line 761

File: README.md [hunk 2]
```diff
@@ -466,6 +445,8 @@ Translations of the guide are available in the following languages:
     good - leading `.` (Option A) and trailing `.` (Option B).
 <sup>[[link](#consistent-multi-line-chains)]</sup>
 
+At Goldstar, we use (Option B)
```
Comment (stephenprater) [inline README.md]: FWIW - Option(A) will indent properly in vim, while Option(B) will not, as I discovered today to my chagrin.

**Comment by rgraff** [comment]
This is the only thing I can think of when talking about style guidelines:

https://www.youtube.com/watch?v=6GMkuPiIZ2k?autoplay=1

Guidelines

**Comment by jaredhoyt** [comment]
@tlehman @therubymug Is this pull request ready to squashed and merged?

**Comment by tlehman** [comment]
@jaredhoyt Probably, we should review this tomorrow

File: README.md [hunk 3]
```diff
@@ -1505,6 +1479,14 @@ condition](#safe-assignment-in-condition).
     tmp = a * 7
     tmp * b / 50
   end
+  
+  #stabby required if there are default arguments to the lambda
```
Comment (jaredhoyt) [inline README.md]: There should be a space after the `#` according to the [comments styleguide](https://github.com/goldstar/ruby-style-guide#hash-space).

**Pull request merged**